latex/headers/sampler.h

```diff
@@ -35,13 +35,16 @@ enum class coordinate_normalization_mode : unsigned int {
 
 class sampler {
  public:
-  sampler(coordinate_normalization_mode normalizationMode, addressing_mode addressingMode,
-          filtering_mode filteringMode);
+  sampler(coordinate_normalization_mode normalizationMode,
+      addressing_mode addressingMode, filterin_mode filteringMode,
```

Comment:
`filtering_mode`

Reply:
I wonder how that even got lost in there. Either way, it is fixed now. Thank you!